specs/WebResourceResponseReceived.md

```diff
@@ -0,0 +1,219 @@
+# Background
+The WebView2 team has been asked for an API to get the response for a web
+resource as it was received and to provide request headers not available when
+`WebResourceRequested` event fires (such as Authentication headers). The
+`WebResourceResponseReceived` event provides such HTTP response representation
+and exposes the request as sent over the wire.
```

Comment:
But also other non-http things that don't necessarily go over a wire, right?

Reply:
Correct. It's the request as committed. Updated.